kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/relation_expression_spec.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// <reference types="Cypress" />
+
+describe("Relation Expression Tests", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("Define 50x50 Relation expression", () => {
+    // Entry point for each new expression
+    cy.ouiaId("expression-container").click();
+
+    // Define new expression as Relation
+    cy.ouiaId("expression-popover-menu").contains("Relation").click({ force: true });
```

Comment:
The `force:true` might hide issues like element not visible, etc. Sometimes can help sth like `.contains("Relation").scrollIntoView().click()`, but depends case by case. There are cases when `force:true` helps for no obvious reason, so just checking.

Reply:
I tried your proposed solution, however I am getting this error in such case.

```
CypressError: Timed out retrying after 4050ms: `cy.click()` failed because this element is not visible:

`<button class="pf-c-menu__item" tabindex="-1" type="button">...</button>`
This element `<button.pf-c-menu__item>` is not visible because its content is being clipped by one of its parent elements, which has a CSS property of overflow: `hidden`, `scroll` or `auto`
Fix this problem, or use `{force: true}` to disable error checking.


```    


The captured screen seems like below:
![Relation Expression Tests -- Define 50x50 Relation expression (failed)](https://user-images.githubusercontent.com/8044780/128351776-b8ad97cb-7921-4d5c-8450-6c41a12c5295.png)

As I am not aware of a bug in "real" application, I prefer to keep `{force: true}`

What do you think?

Reply:
Fair enough, visually this looks correct, feel free to leave the `force:true`.